drivers/tripplite_usb.c

```diff
@@ -467,6 +467,10 @@ void decode_v(const unsigned char *value)
 			  input_voltage_scaled  = 230;
 			  break;
 
+		case 6:   input_voltage_nominal = 
```

Comment:
Is this supposed to be a numeric `6` or a character `'6'`? Nearby precedent leaves me wondering :)

Reply:
From earlier in the thread:
`ups.debug.V: 06 00 0c 01 58 58 0d '....XX.'`

so numeric `6` is likely correct (first byte of `V` response).